R/renderExampleRunPlotSingleCrit.R

```diff
@@ -7,7 +7,7 @@ renderExampleRunPlot.MBOExampleRun = function(object, iter, densregion = TRUE,
   requirePackages("ggplot2")
 
   iters.max = object$control$iters
-  assertIntegerish(iter, lower = 0L, upper = iters.max, len = 1L, any.missing = FALSE)
+  iter = asInt(iter, lower = 1L, upper = iters.max)
```

Comment:
don't confuse renderExampleRun with renderExampleRunPlot :stuck_out_tongue_winking_eye: 
The last one just generates one Plot